keyboards/converter/att56k_usb/info.json

```
@@ -0,0 +1,136 @@
+        {"label":"PF7", "x":9.25, "y":1},
+        {"label":"PF8", "x":10.25, "y":1},
+        {"label":"PF9", "x":11.25, "y":1},
+        {"label":"PF10", "x":12.25, "y":1},
+        {"label":"PF11", "x":13.25, "y":1},
+        {"label":"PF12", "x":14.25, "y":1},
+        {"label":"Attn", "x":0, "y":3},
+        {"label":"Clear", "x":1, "y":3},
+        {"label":"~", "x":2.25, "y":3},
+        {"label":"1", "x":3.25, "y":3},
+        {"label":"2", "x":4.25, "y":3},
+        {"label":"3", "x":5.25, "y":3},
+        {"label":"4", "x":6.25, "y":3},
+        {"label":"5", "x":7.25, "y":3},
+        {"label":"6", "x":8.25, "y":3},
+        {"label":"7", "x":9.25, "y":3},
+        {"label":"8", "x":10.25, "y":3},
+        {"label":"9", "x":11.25, "y":3},
+        {"label":"0", "x":12.25, "y":3},
+        {"label":"-", "x":13.25, "y":3},
+        {"label":"=", "x":14.25, "y":3},
+        {"label":"Back Space", "x":15.25, "y":3, "w":2},
+        {"label":"PA1", "x":17.5, "y":3},
+        {"label":"PA2", "x":18.5, "y":3},
+        {"label":"No", "x":19.5, "y":3},
+        {"label":"Num Lock", "x":20.75, "y":3},
+        {"label":"+", "x":21.75, "y":3},
+        {"label":"'", "x":22.75, "y":3},
+        {"label":"Space", "x":23.75, "y":3},
+        {"label":"Cmd", "x":0, "y":4},
+        {"label":"Cursr Sel", "x":1, "y":4},
+        {"label":"Tab", "x":2.25, "y":4, "w":1.5},
+        {"label":"Q", "x":3.75, "y":4},
+        {"label":"W", "x":4.75, "y":4},
+        {"label":"E", "x":5.75, "y":4},
+        {"label":"R", "x":6.75, "y":4},
+        {"label":"T", "x":7.75, "y":4},
+        {"label":"Y", "x":8.75, "y":4},
+        {"label":"U", "x":9.75, "y":4},
+        {"label":"I", "x":10.75, "y":4},
+        {"label":"O", "x":11.75, "y":4},
+        {"label":"P", "x":12.75, "y":4},
+        {"label":"[", "x":13.75, "y":4},
+        {"label":"\\", "x":14.75, "y":4},
+        {"label":"Return", "x":15.75, "y":4, "w":1.5, "ks":[ [0,0], [1.5,0], [1.5,2], [0.25,2], [0.25,1], [0,1], [0,0] ]},
+        {"label":"Back Tab", "x":17.5, "y":4},
+        {"label":"Insert", "x":18.5, "y":4},
+        {"label":"Delete", "x":19.5, "y":4},
+        {"label":"7", "x":20.75, "y":4},
+        {"label":"8", "x":21.75, "y":4},
+        {"label":"9", "x":22.75, "y":4},
+        {"label":"Tab", "x":23.75, "y":4},
+        {"label":"Jump", "x":0, "y":5},
+        {"label":"Erase EOF", "x":1, "y":5},
+        {"label":"Caps Lock", "x":2.25, "y":5, "w":1.75},
+        {"label":"A", "x":4, "y":5},
+        {"label":"S", "x":5, "y":5},
+        {"label":"D", "x":6, "y":5},
+        {"label":"F", "x":7, "y":5},
+        {"label":"G", "x":8, "y":5},
+        {"label":"H", "x":9, "y":5},
+        {"label":"J", "x":10, "y":5},
+        {"label":"K", "x":11, "y":5},
+        {"label":"L", "x":12, "y":5},
+        {"label":";", "x":13, "y":5},
+        {"label":"'", "x":14, "y":5},
+        {"label":"{", "x":15, "y":5},
+        {"label":"Up", "x":18.5, "y":5},
```

Configurator's UI code also doesn't understand the `ks` key, so our custom is to render ISO Enters as 1.25x2.

```suggestion
        {"label":"Return", "x":16, "y":4, "w":1.25, "h":2},
        {"label":"Up", "x":18.5, "y":5},
```